### DISCUSSION TITLE
Meetings invited look the same as my own meetings

### DISCUSSION THREAD

**Discussion by ericzolf** [kewisch/gdata-provider#345] [Support] [answered]
I just started to use the plug-in instead of the caldav integration and I'm surprised to see that all meetings look alike, as if I would be the owner of all of them: I can view them, edit them, but _not_ accept/decline attendance. And the state of accept/tentative/decline is also not visible in the calendar (green is the gdata-provider version, blue the Caldav one, the short meetings have all be declined):
![image](https://user-images.githubusercontent.com/3105685/147925928-6bdb8c55-6cbb-4ebc-bbf8-95f0d05bede8.png)
Is this really expected behavior or do I miss something?

**Comment by kewisch** [comment, marked as answer]
There may be some issues with how the primary email is determined. Is this the main Google Calendar you are subscribed to (i.e. the one with your name)? This one is good for scheduling, the secondary calendars normally do not work well. With CalDAV I believe you can force it by setting an email address for the calendar.

Would be great if you could enable logging, synchronize, clear logs, send yourself an invitation, synchronize. This way you can see in the logs what the invitation event looks like, which may be telling as to which email addresses are used.

**Reply by ericzolf** [reply]
Sorry for the delay. Yes, this is my primary Google Calendar and the e-mail seems to be the correct/main one (username@corp.com, it's a company managed account). The tests will take a little while because it's my working calendar and I need first to remove the CalDav one, both seem to have some interaction, even if I disable the one or the other (also weird, but different topic), and for this I need a quiet working period.

Thinking about it, it might even be that having both calendars, CalDav _and_ Gdata, configured at the same time is the reason for the weirdness...

**Reply by rushowr** [reply]
Your thoughts on having both of them enabled at the same time seem very likely, Please note, however, that I am not involved in the development of either, rather I am just a user.

**Reply by ericzolf** [reply]
No problem, I switched to "pure" CalDav and gave up (for now) on this plugin. It's not perfect, but at least it's one moving piece less.
